cardboard/settings.py

```diff
@@ -43,6 +43,9 @@
     "cardinality-cardboard.herokuapp.com",
 ]
 
+# For Cardboard Chrome Extension
+CSRF_TRUSTED_ORIGINS = ["chrome-extension://cahmppnjflkbimomgndbcmbdoafdegbi"]
```

Comment:
Is this guaranteed to be the same for everyone? Or is it different per user?

Reply:
This just got published in the Chrome extensions store. The one that everyone will download with have a fixed ID. In development, or if you side load the extension, you will have a different ID. Do you have a problem if we wildcard it instead "chrome-extension://*"?

Reply:
Ok got it. I guess as a bandaid we can do the wildcard, it's sort of a security hole but realistically there's not much to attack.

I think the way this "should" work is with token based auth. We could add oauth login to cardboard ([link](https://django-oauth-toolkit.readthedocs.io/en/latest/rest-framework/getting_started.html)) and then use [chrome.identity](https://developer.chrome.com/docs/extensions/reference/api/identity) in the extension to authenticate with cardboard. 

I think it's probably not actually that much work to implement this but we can punt on it for now and track it in a separate issue.